src/OrchardCore/OrchardCore.DisplayManagement/IShape.cs

```diff
@@ -98,12 +99,12 @@ public static TagBuilder GetTagBuilder(this IShape shape, string defaultTagName
 
             var tagBuilder = new TagBuilder(tagName);
 
-            if (shape.Attributes != null)
+            if (shape.Attributes != null && shape.Attributes.Any())
```

Comment:
`shape.Attributes.Count > 0` too fast so need to slow down with LINQ? 😢

Reply:
@lahma you need to refresh your skills ;) It's generating the same IL these days. So LINQ all the way as it's easier to read IMO.

Reply:
I need to get the compiler/JIT that you are using 😝 

https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzmAPgAQAwAJMCMALANwCwAUDvgQHQAysAjuRZZgMz4BMuAwrgDelXKNwixASQAiUAMYIoAexgQATgE8APIWxwa2AHy4AgggRqowAK4IApiiG4A5nYQlcAXwmjKP/FzASkoANrgAsgQAFACUuAC8xmYWVrYOuACE8bgw1iFhAGQFpuaWNvYotCYwGrGs/py4QaER3LEJSaWpFZnZufm4RSUp5Q60fErWMAi4xtj1FJ5AA==

Reply:
I opened sharplab too when I realized these were on interfaces. You win!

Reply:
🥳

Reply:
@sebastienros even if it is the same thing, I thing `.Count > 0` is nicer and should be used when available.